### ISSUE TITLE
Performance way of using LuaTable.get_Item()

### ISSUE THREAD

**Issue by DangerKiddy** [NLua/NLua#411]
Hi, I'm using NLua for modding in Unity game, and here I have ability to draw things on the screen, using lua, so, each frame lua will call a function which would draw something in C#. Problem is I'm using my own conversations from lua to C#(and vice versa), and conversations such like this one:
```csharp
        AddConversionClr((vectorLua) =>
        {
            LuaTable vec = vectorLua as LuaTable;

            return new Vector3((float)Convert.ToDecimal(vec["x"]), (float)Convert.ToDecimal(vec["y"]));
        });
```
would take a lot of time to proceed (Especially when I'm using drawing with lua)
![image](https://user-images.githubusercontent.com/42438297/110947612-f7183000-8372-11eb-9f4b-b8ece9c350df.png)

I've found that I can convert LuaTable to a Dictionary, but would it be much better to use:
```csharp

        AddConversionClr((vectorLua) =>
        {
            LuaTable vec = vectorLua as LuaTable;
            Dictionary<object, object> tbl = lua.GetTableDict(vec);

            return new Vector3((float)tbl["x"], (float)tbl["y"]);
        });
```
instead of the old one? If not then what I should to do?

**Comment by viniciusjarina** [comment]
I don't this this API belong to NLua `AddConversionClr`

**Comment by DangerKiddy** [comment]
> I don't this this API belong to NLua `AddConversionClr`

Yeah, it's my conversion to metatables. Problem not in it, problem in gettings items of LuaTable in C# like this:
```csharp
LuaTable someTable = // table which C# got from lua
print(someTable["MyCoolValue"])
```
is a bit slow as you can see.
